### DISCUSSION TITLE
Search function not working as intended?

### DISCUSSION THREAD

**Discussion by MaximeAgez** [brightway-lca/brightway-live#42] [Report Limitation]
### Affected Brightway Package

bw2data

### What's broken or not working as expected?

Using the Mobility example database, I am just searching through the database, and the search() functions just finds nothing:

`bw2data.Database('Mobility example').search('car')`

The line above should return me 5 entries but it returns an empty list.

### What's the error message?

```python
No error message popping up.
```

### In order to avoid multiple reports of the same issue, please confirm, that...

- [X] I searched for similar reported limitations and didn't find any.

**Comment by michaelweinold** [comment]
Thank you for reporting! This is related to:

 - [Searchable Databases](https://docs.brightway.dev/projects/live/content/limitations.html#searchable-databases)
